x/poolmanager/types/keys.go

```diff
@@ -33,13 +33,21 @@ var (
 	DenomTradePairPrefix = []byte{0x04}
 
 	// KeyTakerFeeStakersProtoRev defines key to store the taker fee for stakers tracker.
+	// Deprecated: Now utilizes KeyTakerFeeStakersProtoRevArray.
 	KeyTakerFeeStakersProtoRev = []byte{0x05}
 
 	// KeyTakerFeeCommunityPoolProtoRev defines key to store the taker fee for community pool tracker.
+	// Deprecated: Now utilizes KeyTakerFeeCommunityPoolProtoRevArray.
 	KeyTakerFeeCommunityPoolProtoRev = []byte{0x06}
 
 	// KeyTakerFeeProtoRevAccountingHeight defines key to store the accounting height for the above taker fee trackers.
 	KeyTakerFeeProtoRevAccountingHeight = []byte{0x07}
+
+	// KeyTakerFeeStakersProtoRevArray defines key to store the taker fee for stakers tracker coin array.
+	KeyTakerFeeStakersProtoRevArray = []byte{0x08}
+
+	// KeyTakerFeeCommunityPoolProtoRevArray defines key to store the taker fee for community pool tracker coin array.
+	KeyTakerFeeCommunityPoolProtoRevArray = []byte{0x09}
```

Comment:
I could have probably used the old key, but decided on using a new one to just prevent any possible unforeseen bugs.